integration_tests/dbt_project.yml

```diff
@@ -5,7 +5,7 @@ config-version: 2
 
 vars:
   qualtrics_schema: qualtrics_source_integration_tests
-
+ 
```

Comment:
Not a note in particular, but surprised you did not need to add a seed configuration! Did you do anything in particular to avoid the need to add seed configs?

Reply:
i did have to edit one seed file with an `integer` field that was too big/out of range for redshift and postgres, but nothing other than that!